core/src/rpc.rs

```diff
@@ -270,6 +270,53 @@ impl JsonRpcRequestProcessor {
         Ok(new_response(&bank, response))
     }
 
+    pub fn get_multiple_accounts(
+        &self,
+        pubkeys: Vec<Pubkey>,
+        config: Option<RpcAccountInfoConfig>,
+    ) -> Result<RpcResponse<Vec<Option<UiAccount>>>> {
+        let mut accounts: Vec<Option<UiAccount>> = vec![];
+
+        let config = config.unwrap_or_default();
+        let bank = self.bank(config.commitment);
+        let encoding = config.encoding.unwrap_or(UiAccountEncoding::Binary);
+        check_slice_and_encoding(&encoding, config.data_slice.is_some())?;
+
+        for pubkey in pubkeys {
+            let mut response_account = None;
+            if let Some(account) = bank.get_account(&pubkey) {
```

Comment:
Most of this code can't be shared with `get_account_info()`?

Reply:
😳 lazy Tyera